spec/spec_helper_acceptance.rb

```diff
@@ -22,12 +22,18 @@
   c.before :suite do
     # Install module and dependencies
     hosts.each do |host|
-      if fact_on(host, 'osfamily') == 'RedHat'
+      if fact_on(host, 'os.family') == 'RedHat'
         # don't delete downloaded rpm for use with BEAKER_provision=no +
         # BEAKER_destroy=no
         on host, 'sed -i "s/keepcache=.*/keepcache=1/" /etc/yum.conf'
         # refresh check if cache needs refresh on next yum command
         on host, 'yum clean expire-cache'
+
+        major = fact_on(host, 'os.release.major')
```

Comment:
This is different from https://github.com/theforeman/foreman-installer-modulesync/commit/0303d8769398485c1d41accb2a9f7a944707e1f2. If you're going to add this, please ensure we do a full modulesync. It's designed for that. Now you're manually copying over the results.